.eslintrc.js

```diff
@@ -85,7 +86,7 @@ module.exports = {
         'max-lines-per-function': 'error',
         'max-nested-callbacks': 'error',
         'max-params': 'error',
-        'max-statements': 'error',
+        // 'max-statements': 'error',
         'max-statements-per-line': 'error',
```

Comment:
the config in this file should be same as the one provided by microverse [[details here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)]